pkg/auth/auth_handler.go

```diff
@@ -207,6 +212,13 @@ func (a *AuthHandler) isAdmin(username string) (bool, error) {
 		context.Background(), fmt.Sprint(username), CapabilityName, CapabilityType)
 }
 
+func (a *AuthHandler) isClusterOwnedByUser(username string, clusterID string) bool {
```

Comment:
just for readability and keeping convention, can change signature to '(clusterID, username string)'

Reply:
DONE